front/state/action-creators/index.ts

```diff
@@ -31,4 +31,17 @@ export const userLogin = (user: User, callBack: () => void) => {
       dispatch({ type: ActionType.USER_LOGIN_ERROR, payload: error });
     }
   }
+}
+
+export const userLocation = () => {
```

Comment:
이 함수는 유저의 위치정보를 가져오는 함수인가요? 그렇다면 함수명이 아래와 같이 앞부분에 get과같은 동사를 붙여 함수에 대한 의도를 명시적으로 표현하는게 가독성이 좋을듯합니다.
```suggestion
export const getUserLocation = () => {
```

Reply:
넵 알겠습니다~!